### PR TITLE
v7.0.0-beta.3

### PR DESCRIPTION
**Pull request by MBilalShafi**
### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] `yarn && yarn release:build` (make sure you have the latest dependencies installed, and build the packages)
- [ ] `yarn release:publish` (release the versions on npm, you need your 2FA device)
- [ ] `yarn release:tag` (push the newly created tag)

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```bash
git push -f upstream next:docs-next
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-next)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-next` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,62 @@
...
+
+Same changes as in `@mui/x-data-grid@7.0.0-beta.3`.
+
+#### `@mui/x-data-grid-premium@7.0.0-beta.3` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.0.0-beta.3`.
+
+### Charts / `@mui/x-charts@7.0.0-beta.3`
+
+- [charts] Add Gauge component (#11996) @alexfauquette
```
Comment (flaviendelangle) [inline CHANGELOG.md:39]: This one is worth highlighting :ok_hand:

**Review by LukasTy — APPROVED** [review]
LGTM other than the Gauge component worth highlighting. 👍 
cc @alexfauquette

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,62 @@
...
 
+## v7.0.0-beta.3
+
+_Feb 16, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Charts get a [built in grid](https://next.mui.com/x/react-charts/axis/#grid)
+  ![image](https://github.com/mui/mui-x/assets/45398769/74299f54-f020-4135-b38c-dc88a230db30)
+- 🐞 Bugfixes
```
Comment (alexfauquette) [inline CHANGELOG.md:14]:
```suggestion
- 🎛️ Charts get a [Gauge component](https://next.mui.com/x/react-charts/gauge/) useful to display a value in a dashboard.
  ![image](https://github.com/mui/mui-x/assets/45398769/fb7a94b5-bef6-4fc2-a0cd-d6ff5b60fa8b)
- 🐞 Bugfixes
```

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (LukasTy) [reply CHANGELOG.md:14]: Would it also make sense to highlight the addition of `ChartsGrid`? 🤔

Comment (MBilalShafi) [reply CHANGELOG.md:14]: Isn't it the same as the first highlight?

Comment (LukasTy) [reply CHANGELOG.md:16]: 🤦 
Forget what I said... The diff confused me...

**Pull request merged**